client/packages/common/src/utils/environment/EnvUtils.ts

```diff
@@ -53,9 +58,23 @@ const mapRoute = (route: string): RouteMapping => {
   }
 };
 
+const getPlatform = () => {
+  // 'Mozilla/5.0 (Linux; Android 8.1.0; Lenovo TB-X304L Build/OPM1.171019.026; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/102.0.5005.78 Safari/537.36'
```

Comment:
I left in the example of an android user agent.. just in case it helps someone later 🤷